test/unit/spec/v2/ion/responseTransformer_spec.js

```diff
@@ -515,6 +515,17 @@ describe('v2/ion/responseTransformer', function() {
             type: 'APPLE',
             name: 'redirect-idp',
           },
+          {
+            action: jasmine.any(Function),
```

Comment:
Let's try to avoid jasmine matchers in favor of Jest:

```suggestion
            action: expect.any(Function),
```